.github/ISSUE_TEMPLATE.md

```diff
@@ -1,19 +0,0 @@
-<!-- Delete non-relevant parts if this is not a bug report -->
```

Comment:
As we are required to document our process quite heavily the issue template was cumbersome. As we are still adding commits we will revert this commit when we put this pull request on `READY`.